### DISCUSSION TITLE
WinForms - Adding ChromiumWebBrowser using designer crashes with connection lost error

### DISCUSSION THREAD

**Discussion by goophps** [cefsharp/CefSharp#4018] [General]
![image](https://user-images.githubusercontent.com/49714224/155124355-33328da7-125d-4c24-9462-74fc565e2d3c.png)

**Comment by goophps** [comment]
This prompt appears when I click save after inserting the control into the form.

**Reply by amaitland** [reply]
Converting to discussion as no bug report was filled out.

English only please. Provide a list of detailed steps to reproduce the problem, version numbers etc.

**Reply by goophps** [reply]
1 My development environment: vs2022 and .net6
2 I installed CefSharp.WinForms.NETCore 98.1.210
The installation is successful, everything is normal, when I drag and drop the control from the toolbox to the form and click save, an error is reported, prompting "The connection to the server has been lost"

**Reply by amaitland** [reply]
Thanks, much clearer. 

Designer support is currently broken, add the ChromiumWebBrowser instance in code behind for now.

**Reply by goophps** [reply]
Can you fix this problem?

**Reply by amaitland** [reply]
It's unlikely there's a quick fix for this. VS2022 uses the new OOP designer for .Net 6, explicit support is likely required. Issue #4019 will track adding support. A pull request to add support would be welcomed. 

Use code behind as workaround until support added.
